tests/data/power_flow/asym-line/rmatrix-with-n-xmatrix-with-n-c1-c0/params.json

```
@@ -0,0 +1,5 @@
+{
+  "calculation_method": ["newton_raphson"],
+  "rtol": 1e-03,
+  "atol": 1e-03
```

I think its better to aim for an rtol of 1e-8 as our modelling is quite close to that of Open DSS
```suggestion
  "rtol": 1e-08,
  "atol": 1e-06
```